test/fixtures/v2/documentDidSigned.ts

```diff
@@ -0,0 +1,69 @@
+export const documentDidSigned: any = {
+  version: "https://schema.openattestation.com/2.0/schema.json",
+  data: {
+    id: "c4deaa44-b58a-4c9f-b66b-30c0c28f81c9:string:SGCNM21566325",
+    $template: {
+      name: "0b0220dc-e16e-4ef4-8a4c-7ffd021d9c58:string:CERTIFICATE_OF_NON_MANIPULATION",
+      type: "7247b6c5-0bed-4393-ac85-a97482ae208a:string:EMBEDDED_RENDERER",
+      url: "1b8fe35e-c338-4061-bb22-17a791ad06b8:string:https://demo-cnm.openattestation.com",
+    },
+    issuers: [
+      {
+        name: "48c17918-af33-4578-a0ce-0efbcad6a088:string:DEMO STORE",
+        revocation: { type: "15c54c38-d2eb-4c5d-adf9-41f028be1f89:string:NONE" },
+        identityProof: {
+          type: "4d9787d9-05f5-48fe-9e4f-8d8577c1612d:string:DID",
+          id: "859aae23-cf04-45bb-b687-43e4d20b0790:string:did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89",
+          purpose: "7f2b206e-1113-430f-b059-19cc9c8b5694:string:DOCUMENT_ISSUANCE",
```

Comment:
Will multiple types for `purpose` be supported?

Reply:
hmm actually in this case where we have separated out the types, this purpose can be made redundant.